packages/app/src/static-props/variants/get-variant-chart-data.ts

```diff
@@ -37,6 +38,7 @@ export function getVariantChartData(variants: NlVariants | undefined) {
       [`${firstVariant.name}_occurrence`]: value.percentage,
```

Comment:
unrelated to the PR but I just noticed that we're writing value.percentange to the _occurencee property, looks like a bug @VWSCoronaDashboard1 
```suggestion
      [`${firstVariant.name}_occurrence`]: value.percentage,
```